yarn.lock

```diff
@@ -13643,7 +13662,17 @@ minimist-options@^3.0.1:
     arrify "^1.0.1"
     is-plain-obj "^1.1.0"
 
-minimist@0.0.8, minimist@1.1.x, minimist@^1.1.0, minimist@^1.1.1, minimist@^1.1.3, minimist@^1.2.0, minimist@^1.2.5:
+minimist@0.0.8:
```

Comment:
https://github.com/advisories/GHSA-vh95-rmgr-6w4m
You should enforce the usage of recent version